src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -132,7 +132,20 @@
                                 [messages]="messages"></cd-select-badges>
             </span>
           </div>
+        </div>
 
+        <!-- Enabled -->
+        <div class="form-group row">
+          <label class="col-sm-3 col-form-label"
+                 i18n>Enabled</label>
+          <div class="col-sm-9">
+						<div class="btn-group" btnRadioGroup formControlName="enabled">
+							<label [btnRadio]="true" class="btn btn-primary"
+									 tabindex="0" role="button">Enabled</label>
+							<label [btnRadio]="false" class="btn btn-primary"
+										 tabindex="0" role="button">Disabled</label>
```

Comment:
Please also format the other attributes 1 per line.
```suggestion
										 tabindex="0"
										 role="button"
										 i18n>Disabled</label>
```